crates/service-protocol/src/message/header.rs

```diff
@@ -112,6 +121,9 @@ const SET_STATE_ENTRY_MESSAGE_TYPE: u16 = 0x0801;
 const CLEAR_STATE_ENTRY_MESSAGE_TYPE: u16 = 0x0802;
 const CLEAR_ALL_STATE_ENTRY_MESSAGE_TYPE: u16 = 0x0803;
 const GET_STATE_KEYS_ENTRY_MESSAGE_TYPE: u16 = 0x0804;
+const GET_PROMISE_ENTRY_MESSAGE_TYPE: u16 = 0x0808;
```

Comment:
There was another API for having "streams" that I designed and just temporarily removed, because I won't have time to finish it now. but it's coming soon, and i forgot to re-enumerate the numbers here :)